scripts/deployment/main.py

```diff
@@ -58,11 +58,11 @@ def get_commit_id() -> str:
 
 def collect_contracts_info(contracts: dict[str, DeployedContract]) -> dict:
     return {
-        name: dict(
-            address=contract.address,
-            deployment_block=contract.deployment_block,
-            deployment_args=contract.deployment_args,
-        )
+        name: {
+            "address": contract.address,
+            "deployment_block": contract.deployment_block,
+            "deployment_args": contract.deployment_args,
+        }
```

Comment:
Please drop this change, this is a case where the dict keywords actually make more sense, ergonomically speaking.